html/search.js

```
@@ -20,25 +20,29 @@ async function highlightResults(node) {
-        await highlightResults(evt.detail.elt);
-        break;
-    }
-  });
-}
-
 function setSearch(s) {
   let el = document.getElementById("search");
   el.value = s;
 }
+
+document.addEventListener("DOMContentLoaded", (event) => {
```

ESLint

[error] 'event' is defined but never used.

(no-unused-vars)